pro/src/screens/OfferType/OfferTypeButton/OfferTypeButton.tsx

```diff
@@ -0,0 +1,36 @@
+import cn from 'classnames'
+import React, { FunctionComponent, SVGProps } from 'react'
+
+import { ReactComponent as TickIcon } from 'icons/tick.svg'
+
+import styles from './OfferTypeButton.module.scss'
+
+interface IOfferTypeButton {
+    isSelected: boolean;
+    Icon: FunctionComponent<SVGProps<SVGSVGElement> & { title?: string | undefined; }>;
+    label: string;
+    onClick?: () => void;
+    className?: string;
+}
+
+const OfferTypeButton = ({
+  isSelected,
+  Icon,
+  label,
+  onClick,
+  className,
+}: IOfferTypeButton): JSX.Element => (
+  <button
+    className={cn(styles.button, { [styles['is-selected']]: isSelected }, className)}
+    onClick={onClick}
+    type='button'
+  >
+    {isSelected ? <TickIcon className={styles['button-checked']} /> : null}
+    <Icon className={styles['button-icon']} />
+    <p className={styles['button-label']}>
```

Comment:
il est invalide au W3C de mettre une balise de type bloc dans une balise inline. (à l'exeption de `<a>` je crois)
```suggestion
    <span className={styles['button-label']}>
```

mais du coup si tu utilises un radio : 


```suggestion
    
    <label>
       <input type radio>
       <Icon/>
       <span/>
    </label>
```